### PR TITLE
Update ez_sql_srv.php

### PR DESCRIPTION
**Pull request by xamount**
Set connection options to pull date fields as a string rather than a Date class.

### CODE REVIEW

File: sqlsrv/ez_sql_sqlsrv.php [hunk 1]
```diff
@@ -98,7 +98,7 @@ function connect($dbuser='', $dbpassword='', $dbname='', $dbhost='localhost')
 				$connectionOptions["UID"] = $dbuser;
 				$connectionOptions["PWD"] = $dbpassword;
 			}
-//			$connectionOptions = array("UID" => $dbuser, "PWD" => $dbpassword, "Database" => $dbname);
+//			$connectionOptions = array("UID" => $dbuser, "PWD" => $dbpassword, "Database" => $dbname, "ReturnDatesAsStrings" => true);
```
Comment (xamount) [inline sqlsrv/ez_sql_sqlsrv.php]: I've uncommented it

**Comment by gnowxilef** [comment]
have you tested that this works?

**Comment by xamount** [comment]
Yes I have tested it

**Comment by gnowxilef** [comment]
maybe allow this option to be configurable.

**Comment by techno-express** [comment]
Issue resolved with latest release.

**Pull request closed without merging**